addons/web/static/src/core/dropzone/dropzone_hook.js

```diff
@@ -16,9 +16,15 @@ export function useDropzone(targetRef, onDrop, extraClass, isDropzoneEnabled = (
     useExternalListener(document, "dragleave", onDragLeave, { capture: true });
     // Prevents the browser to open or download the file when it is dropped
     // outside of the dropzone.
-    useExternalListener(window, "dragover", (ev) => ev.preventDefault());
+    useExternalListener(window, "dragover", (ev) => {
+        if(dragCount){
```

Comment:
If you're working with JS files, I highly encourage you to use the formatting/linting tools provided in web to avoid formatting mistakes like this one.
```suggestion
        if (dragCount) {
```